--- mandriva_pptp/ppas.sh.old	2010-03-26 15:10:54.000000000 +0300
+++ mandriva_pptp/ppas.sh	2010-05-01 10:30:10.563517545 +0400
@@ -7,6 +7,6 @@
 OFS=$IFS
 IFS="
 "
-/usr/bin/ld -b elf32-i386 -m elf_i386  --dynamic-linker=/lib/ld-linux.so.2    -L. -o vpnpptp link.res
+/usr/bin/ld -b elf32-x86_64 -m elf_x86_64  --dynamic-linker=/lib/ld-linux.so.2    -L. -o vpnpptp link.res
 if [ $? != 0 ]; then DoExitLink vpnpptp; fi
 IFS=$OFS
